### PR TITLE
Download correct installer version

### PR DESCRIPTION
**Pull request by dhensby**
Download the correct installer version for a given core release

### CODE REVIEW

File: travis_setup.php [hunk 1]
```diff
@@ -128,7 +140,20 @@
 
 run("tar -cf $moduleArchivePath * .??*");
 
-run("git clone --depth=100 --quiet -b $coreInstallerBranch git://github.com/silverstripe/silverstripe-installer.git $targetPath");
+run("wget $installerDist -O /tmp/installer.zip");
```
Comment (tractorcow) [inline travis_setup.php]: http://php.net/manual/en/function.sys-get-temp-dir.php please :D

File: travis_setup.php [hunk 2]
```diff
@@ -128,7 +140,20 @@
...
+$folderName = $zip->getNameIndex(0);
+
+run("unzip /tmp/installer.zip -d /tmp/");
+
+run("mkdir -p $targetPath");
+
+run("mv /tmp/{$folderName}/* $targetPath");
+run("mv /tmp/{$folderName}/.* $targetPath");
+
+die;
```
Comment (tractorcow) [inline travis_setup.php]: `git clone --branch` can take a tag. It would be a lot less code than all of this.

File: travis_setup.php (hunk 1 above)
Comment (dhensby) [inline travis_setup.php]: done

File: travis_setup.php (hunk 2 above)
Comment (dhensby) [inline travis_setup.php]: removing the `die` would be a good idea...

**Pull request merged**